docs/UserGuide.md

```diff
@@ -87,132 +87,170 @@ Shows a message explaining how to access the help page.
 
 Format: `help`
 
-### Adding a new activity: `add t/activity`
+### Adding a new patient's details: `add p/patient`
 
-Adds a new activity into your GoMedic scheduler.
+Adds a new patient into the GoMedic application.
 
-Format: `add t/activity s/START_TIME e/END_TIME ti/TITLE [d/DESCRIPTION]`
+Format: `add p/patient n/NAME a/AGE g/GENDER h/HEIGHT w/WEIGHT b/BLOOD_TYPE c/CONTACT_NUMBER o/[OPTIONAL...]`
```

Comment:
Shall we change p/patient to t/patient (explicit check for t => so later we can just get the flag for t and check whether t is activity, patient or doctor)